operator/hack/check-running-on-kind.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
What is this for?

Reply:
If you see the VSCode launch config for running the operator in an IDE and the check task - its used by my launch config to prevent a dev from accidentally running the Paladin operator against an unintended cluster. As an SRE working across a lot of clusters daily, I appreciate when projects have safeguards in place for local development.